### ISSUE TITLE
Thanks & C# version

### ISSUE THREAD

**Issue by bzier** [iheartradio/open-m3u8#60]
Hi, I just wanted to say thank you for making this library available. It's always nice to see people (especially companies) giving back to the open source community.

I also wanted to let you know that I've forked this project and translated it into C#, targeting .NET Standard 2.0. I've tried to provide attribution and references back to this repository. Feel free to take a look and let me know if you'd like to see any more/different attribution.

https://github.com/bzier/open-m3u8

Thanks again,
-Brian

**Comment by trevor403** [comment]
This is so lovely, that's exactly what the opensource community is about! I just checked out your repo (pun intended) good stuff man!

**Comment by Wopple** [comment]
Hello Brian,

I'm glad you are finding this library useful. It may not be tenable to maintain parity between two languages in two different projects, so it may be better for you to think of your code base as your own.

**Comment by bzier** [comment]
Agreed, I don't expect to keep things in sync. That certainly wouldn't be easy. Just going through the translation itself was a time consuming exercise. I was mostly just wanting to make sure I give credit where credit is due, rather than set any expectation for maintaining the two libraries together.
